db/migrate/20181126133537_assign_existing_exports_to_atos.rb

```diff
@@ -0,0 +1,19 @@
+class AssignExistingExportsToAtos < ActiveRecord::Migration[5.2]
```

Comment:
records in the exports table need to have the external_system_id set (enforced in later migration). This migration fixes existing data